lib/rules/web/admin_console/4.4/overview.xyaml

```diff
@@ -6,11 +6,12 @@ wait_overview_items_loaded:
 check_list_heading_shown:
   element:
     selector:
-      xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<heading>']
+      xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<heading>' and not(contains(@class,'pf-m-disabled'))]
 check_list_item:
   elements: 
   - selector: &overview_item
       xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<resource_name>']/ancestor::div[contains(@class,'project-overview__item--<resource_kind>')]
+    timeout: 90
```

Comment:
Do we need so long time here?

Reply:
Yes, it's loading for a long time on my local side. Parallel running jobs may be also slow.